components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3763,6 +3763,9 @@ def getUnionTypeTemplateVars(type, descriptorProvider):
     elif type.isPrimitive():
         name = type.name
         typeName = builtinNames[type.tag()]
+    elif type.isByteString():
+        name = type.name
+        typeName = "ByteString"
```

Comment:
Makes more sense to move these 3 lines up to align with the other String types, no?